x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v8_4_0.test.ts

```diff
@@ -55,17 +57,18 @@ describe('8.4.0 Endpoint Package Policy migration', () => {
     };
   };
 
-  it('adds advanced file monitoring defaulted to false', () => {
+  it('adds advanced file monitoring defaulted to false and ensures crendential hardening is added and false.', () => {
```

Comment:
@magermark yes - I assumed that's what we wanted since we generally keep new features disabled when migrating old policies.  It's not hard to change it be on in migration, I just need the official word.